### ISSUE TITLE
Windows: Events channel for Powershell events is incorrect case. This leads to dropped events.

### ISSUE THREAD

**Issue by ExceptionalHandler** [facebook/osquery#4538]
```
+--------------------+------------+----------+-------------+
| osquery_version    | os_version | os_build | os_platform |
+--------------------+------------+----------+-------------+
| 3.2.5-22-ga4f1bedc | 10.0.17134 | 17134    | windows     |
+--------------------+------------+----------+-------------+
```

### What have you already tried?
After enabling powershell script block logging from Gpedit.msc and registry, and running some powershell commands, I see event ID 4104 in Windows Event Viewer under Microsoft-Windows-PowerShell/Operational section. 
 
### What did you expect to see?
Expect to see some events in powershell_events table. 

### What did you see instead?
powershell_events table is empty. 
Osquery is running and subscribing to Windows Events correctly. The Windows_events table has recent windows events from the respective channels.

**Comment by muffins** [comment]
Interesting, this definitely looks like a problem, however we have been actively ingesting Powershell from this table for a bit and we are getting back powershell script logs, so I think there might be something else at play here. I changed the code logic to see why we are firing with these case differences, but I can still see instances where we don't fire even with correct casing.

**Comment by muffins** [comment]
This was fixed in #4550
